btr-api/src/btr_api/resources/request.py

```diff
@@ -285,3 +288,24 @@ def get_all_comments(req_uuid: str):  # pylint: disable=redefined-builtin
         return exception_response(aex)
     except Exception as exception:  # noqa: B902
         return exception_response(exception)
+
+
+@bp.route('/auto_reject', methods=('POST',))
+def auto_reject_requests():
+    """Auto reject requests."""
+    days_of_awaiting = current_app.config.get('DAYS_TO_AUTO_REJECT_REQUESTS', 60)
```

Comment:
this should be set in the config. Right now it'll always just do the default and we'd have to make code changes to change it. When its specified in the config then we can redeploy with a new env value